Mathlib/Analysis/Calculus/Deriv/Basic.lean

```diff
@@ -638,15 +638,27 @@ variable (c : F) (s x L)
 theorem hasDerivAtFilter_const : HasDerivAtFilter (fun _ => c) 0 x L :=
   (hasFDerivAtFilter_const c x L).hasDerivAtFilter
 
+theorem hasDerivAtFilter_zero : HasDerivAtFilter (0 : 𝕜 → F) 0 x L := by
+  simp [hasDerivAtFilter_iff_isLittleO]
```

Comment:
```suggestion
theorem hasDerivAtFilter_zero : HasDerivAtFilter (0 : 𝕜 → F) 0 x L :=
  hasDerivAtFilter_const _ _
```
etc, to ensure that the proof doesn't need a different level of generality.

Reply:
Good point, done.